frontend/src/main/java/com/java/academy/gym/dto/LocaleUserMessagesRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package com.java.academy.gym.dto;
+
+import java.util.List;
+
+public class LocaleUserMessagesRequestDto {
```

Comment:
To be honest I try avoid creating  separated DTO objects and prefer to use the entities, because you keep all things in one place so you get readable, maintainable code without repetition. You can use annotations like @JsonIgnore, @Transient to make it.

Reply:
In this case we need only language code and messages' keys data in request - so instead of this we should accept whole Locale object and list of UserMessage objects?

Reply:
yes, and you can use the UserMessage as a DTO object you don't need create LocaleUserMessagesRequestDto